kgraphql/src/test/kotlin/com/apurebase/kgraphql/specification/typesystem/ScalarsSpecificationTest.kt

```diff
@@ -21,6 +22,71 @@ import java.util.UUID
 @Specification("3.1.1 Scalars")
 class ScalarsSpecificationTest {
 
+    @Test
+    fun `built-in scalars should be available by default`() {
+        val schema = KGraphQL.schema {
+            query("int") {
+                resolver<Int> { 1 }
+            }
+            query("float") {
+                resolver<Float> { 2.0f }
+            }
+            query("double") {
+                resolver<Double> { 3.0 }
+            }
+            query("string") {
+                resolver<String> { "foo" }
+            }
+            query("boolean") {
+                resolver<Boolean> { true }
+            }
+            // TODO: ID, cf. https://github.com/stuebingerb/KGraphQL/issues/83
+        }
+
+        val response = deserialize(schema.executeBlocking("{ int float double string boolean }"))
+        assertThat(response.extract<Int>("data/int"), equalTo(1))
+        assertThat(response.extract<Float>("data/float"), equalTo(2.0))
```

Comment:
IMHO this should be `equalTo(2.0f)` but haven't found out why that doesn't work.